orttraining/orttraining/python/training/ortmodule/torch_cpp_extensions/__init__.py

```diff
@@ -13,6 +13,11 @@
 import threading
 from functools import wraps
 from onnxruntime.capi import _pybind_state as C
+from torch.onnx import register_custom_op_symbolic
+from ._aten_op_factory import ATenOpFactory
+
+
+_onnx_opset_version = 1
```

Comment:
This information is available at https://github.com/microsoft/onnxruntime/blob/master/orttraining/orttraining/python/training/ortmodule/__init__.py#L16

In fact, global configuration should be in a single central spot as opposed to spread throughout the code

Why `1` and not `12`?

Reply:
Just follow here to https://github.com/microsoft/onnxruntime/blob/bcd50afafbb95d9206535802669a7b4a9ec5a019/tools/python/register_custom_ops_pytorch_exporter.py#L11 to use 1 instead of other version.